src/govuk/components/character-count/character-count.test.js

```diff
@@ -321,6 +322,207 @@ describe('Character count', () => {
         })
       })
     })
+
+    describe('JavaScript configuration', () => {
+      let examples
+      beforeAll(() => {
+        examples = getExamples('character-count')
+      })
+
+      describe('at instanciation', () => {
```

Comment:
Typo (I struggle with this word too!)

```suggestion
      describe('at instantiation', () => {
```